Gemfile

```diff
@@ -18,6 +18,7 @@ gem 'redcarpet'
 gem 'sass-rails'
 gem 'snappconfig'
 gem 'uglifier'
+gem 'whenever', require: false
```

Comment:
That's what the docs told me to do. My understanding is limited, but you require: false whenever you have something that runs from the command line or as a cron job but is not a library used in the code itself.